games-engines/qtads/qtads-3.1.0.ebuild

```
@@ -0,0 +1,43 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit qmake-utils xdg
+
+DESCRIPTION="Multimedia interpreter for TADS text adventures"
+HOMEPAGE="https://realnc.github.io/qtads"
+SRC_URI="https://github.com/realnc/qtads/releases/download/v${PV}/${P}-source.tar.xz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+sound"
+
+BDEPEND="
+	virtual/pkgconfig
+"
+DEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5[ssl]
+	dev-qt/qtwidgets:5
+	sound? (
+		media-libs/libsdl2[sound]
+		media-libs/libsndfile
+		media-libs/libvorbis
+		media-sound/fluidsynth:0=
+		media-sound/mpg123
+	)
+"
+RDEPEND=${DEPEND}
+
+src_configure() {
+	eqmake5 \
+		PREFIX="${EPREFIX}/usr" \
+		$(usex !sound CONFIG+=disable-audio '') \
```

Trivial but should be updated for EAPI-8 if bumping
```suggestion
		$(usev !sound CONFIG+=disable-audio) \
```